modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -45,11 +45,27 @@ class SeqexecEngine(settings: SeqexecEngine.Settings) {
 
   def requestRefresh(q: engine.EventQueue): Task[Unit] = q.enqueueOne(Event.poll)
 
-  def eventProcess(q: engine.EventQueue): Process[Task, SeqexecEvent] =
+  def eventProcess(q: engine.EventQueue): Process[Task, SeqexecEvent] = {
+
+    implicit def toSequenceState(st: engine.Status): SequenceState = st match {
+      case engine.Status.Waiting   => SequenceState.Idle
+      case engine.Status.Completed => SequenceState.Completed
+      case engine.Status.Running   => SequenceState.Running
```

Comment:
Previously it was calculated when any `Action` under the `Step`s of a `Sequence` errored but now we are using the engine `status` for each `Sequence` which is just `Waiting` when there is an `error` or a normal `pause`, from the Engine perspective it has to do the same in both cases.

So we have to either check whether there is an error when *printing* the `status` or create another `Engine` `Error` status which would have to set explicitly and handled like `Waiting` by the `engine`.

In either case I think that would need a separate PR.